net-misc/dukto/dukto-6.0.0.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="Easy and multi-platform file transfer tool"
+DESCRIPTION="Dukto is an easy file transfer tool designed for LAN use. \
+Transfer files from one PC (or other device) to another, without worrying \
+about users, permissions, operating systems, protocols, clients, servers and \
+so on… Just start Dukto on the two PCs and transfer files and folders by \
+dragging onto it’s window. That’s all."
+HOMEPAGE="https://www.msec.it/blog/dukto/"
+COPYRIGHT="2011 Emanuele Colombo"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://codeload.github.com/ttys3/dukto/zip/00301800b3a645b80361a59de911839bf49ee32e"
+CHECKSUM_SHA256="599e808fb9206f24386afb32ee96766797d4354b5f34838d8b4dae65f446fe5a"
+SOURCE_DIR="dukto-00301800b3a645b80361a59de911839bf49ee32e"
+PATCHES="dukto-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	dukto$secondaryArchSuffix = $portVersion
+	app:Dukto$secondaryArchSuffix = $portVersion
```

Comment:
Haven't checked the build (yet), but secondaryArchSuffix shouldn't be needed here:
```diff
- 	app:Dukto$secondaryArchSuffix = $portVersion
+ 	app:Dukto
```